src/lib/packet.h

```diff
@@ -377,6 +377,7 @@ typedef enum {
                                * IETF-S/MIME) */
     PGP_PKA_EDDSA = 22,       /* EdDSA from draft-ietf-openpgp-rfc4880bis */
 
+    PGP_PKA_SM2_ENCRYPT = 98, /* SM2 encryption */
```

Comment:
@dewyatt That definitely all makes sense to me. FTR I'm not strongly in favor of one approach vs the other. @ronaldtse your take?